i18n/en.pot

```diff
@@ -51,6 +51,13 @@ msgstr ""
 msgid "Data Entry"
 msgstr ""
 
+msgid ""
+"This action is going to unaproved all the periods. You will have to approve "
+"the data again on the Data Approval section. Are you sure you want to apply "
+"the current values to all months? The existing data in other months will be "
+"overwritten."
+msgstr ""
```

Comment:
The message is not the same than in the task, confirm with @adrianq

Reply:
Yes, it is fine. I am pretty sure the client will change it once he reviews it